code/src/components/Loading.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+
+import "./loading.css";
+
+const Loading = () => {
+  return (
+    <>
```

Comment:
No need for React fragment (`</>`) if you return only one element (in your case `svg`)